src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlConnectionStringBuilderTest.cs

```diff
@@ -147,7 +147,10 @@ private void ExecuteConnectionStringTests(string connectionString)
                 Assert.Equal(valueBuilder1, valueBuilder2);
                 Assert.True(builder2.ContainsKey(key));
             }
+            using (SqlConnection connection = new SqlConnection(connectionString))
+            {
+                Assert.NotNull(connection);
+            }
```

Comment:
@ErikEJ Creating `SqlConnection` instance captures the issue, that wasn't done before.